src/EventListener/ExceptionListener.php

```diff
@@ -7,40 +7,108 @@
     $exitCode = $event->getExitCode();
     $error = $event->getError();
     $errorMessage = $error->getMessage();
+    $io = new SymfonyStyle($this->input, $this->output);
 
     // Make OAuth server errors more human-friendly.
     if ($error instanceof IdentityProviderException && $error->getMessage() === 'invalid_client') {
-      $newErrorMessage = 'Your Cloud Platform API credentials are invalid. Run acli auth:login to reset them.';
+      $new_error_message = 'Your Cloud Platform API credentials are invalid.';
+      $io->comment("Run <options=bold>acli auth:login</> to reset your API credentials.");
     }
 
     if ($error instanceof RuntimeException) {
       switch ($errorMessage) {
         case 'Not enough arguments (missing: "environmentId").':
-          $newErrorMessage = $errorMessage . PHP_EOL . 'Note: "environmentId" can also be an Acquia sitegroup and environment construct, e.g. myapp.dev.';
+        case 'Not enough arguments (missing: "environmentUuid").':
+          $this->writeSiteAliasHelp($io);
+          break;
+        case 'Not enough arguments (missing: "applicationUuid").':
+          $this->writeApplicationAliasHelp($io);
+          break;
+      }
+    }
+
+    if ($error instanceof AcquiaCliException) {
+      switch ($errorMessage) {
+        case '{applicationUuid} must be a valid UUID or application alias.':
+          $this->writeApplicationAliasHelp($io);
+          break;
+        case '{environmentId} must be a valid UUID or site alias.':
+        case '{environmentUuid} must be a valid UUID or site alias.':
+          $this->writeSiteAliasHelp($io);
+          break;
       }
     }
 
     if ($error instanceof ApiErrorException) {
       switch ($errorMessage) {
         case "There are no available Cloud IDEs for this application.\n":
-          $newErrorMessage = $errorMessage . "Delete an existing IDE (acli ide:delete) or contact your Account Manager or Acquia Sales to purchase additional IDEs.\n You may also submit a support ticket to ask for more information (https://insight.acquia.com/support/tickets/new?product=p:ride)";
+          $io->comment("Delete an existing IDE via <options=bold>acli ide:delete</> or contact your Account Manager or Acquia Sales to purchase additional IDEs.");
+          $this->writeSupportTicketHelp($io);
           break;
         default:
-          $newErrorMessage = 'Cloud Platform API returned an error: ' . $errorMessage;
+          $new_error_message = 'Cloud Platform API returned an error: ' . $errorMessage;
       }
+      $io->comment("You can learn more about Cloud Platform API at <href=https://docs.acquia.com/cloud-platform/develop/api/>docs.acquia.com</>");
     }
 
-    if (isset($newErrorMessage)) {
-      $event->setError(new AcquiaCliException($newErrorMessage, [], $exitCode));
+    $io->comment("You can find Acquia CLI documentation at <href=https://docs.acquia.com/acquia-cli/>docs.acquia.com</>");
```

Comment:
Are you not missing surrounding quotes for the hyperlink? It's not clickable for me here.